docs/fsharp-cheatsheet.md

```diff
@@ -125,33 +138,56 @@ See [Literals (MS Learn)](https://learn.microsoft.com/en-us/dotnet/fsharp/langua
 
 ## Functions
 
-The `let` keyword also defines named functions.
+Use the `let` keyword to define named functions.
 
     let negate x = x * -1
     let square x = x * x
-    let print x = printfn "The number is: %d" x
+    let print x = printfn $"The number is: {x}"
+
+### Pipe and Composition Operators
 
-### Pipe and composition operators
 Pipe operator `|>` is used to chain functions and arguments together. Double-backtick identifiers are handy to improve readability especially in unit testing:
```

Comment:
the double backtick stuff is now a complete non sequiter - the point here should be to convey wth a pipe does
and then you have a one liner with an example that says "and `>>` is the same thing but leaves the argument open". So I'd suggest naming the first fn `squareNegateThenPrint` and the second one `squareNegateThenPrint2` to convey they are almost the same (minus the arg having a name)